operation-payment.x

```diff
@@ -106,6 +106,8 @@ enum PaymentResultCode
     DESTINATION_ACCOUNT_NOT_FOUND = -14,
     //: Amount precision and asset precision are mismatched
     INCORRECT_AMOUNT_PRECISION = -15
+    //: Too much signs in subject
+    TOO_MUCH_SIGNS = -16,
```

Comment:
just imagine that you are front-end developer, and when you post tx with payment op, the result is returned with code TOO_MUCH_SIGNS which you never seen before, will you understand reason of failed result?